doc/source/intro.rst

```diff
@@ -10,26 +10,40 @@ the same time, Pypiper provides immediate advantages over a
 simple shell script.
 
 Pypiper is an example of a simple  `bioinformatics pipeline framework
-<http://databio.org/pipeline_frameworks/>`_.
+<http://databio.org/pipeline_frameworks/>`_. It differs from existing frameworks in its focus on **simplicity** and **sequential pipelines**. 
+To employ pypiper, you will just take your bash script and pass those commands through ``PipelineManager.run()``. This will give you automatic restartability, process monitoring for memory use and compute time, pipeline status monitoring, copious log output, robust error handling, easy debugging tools, guaranteed file output integrity, and a bunch of useful pipeline development helper functions.
 
-Many bioinformatics pipelines are written by students or technicians who don't have time to learn a full-scale pipelining framework, so they just end up using simple bash scripts to piece together commands. Pypiper tries to give 80% of the benefits of a professional-scale pipelining system while requiring very little additional effort.
+A simple example pipeline
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To demonstrate the simplicity, take a look at a very simple but complete pipeline:
+
+.. literalinclude:: ../../example_pipelines/hello_pypiper.py
 
-Just take your bash script and pass those commands through ``PipelineManager.run()`` and you will get automatic restartability, process monitoring for memory use and compute time, pipeline status monitoring, copious log output, robust error handling, easy debugging tools, guaranteed file output integrity, and a bunch of useful pipeline development helper functions.
+There's nothing complex here: we are choosing an output folder, and running a single command, much like you may do in a shell script. Building pypiper pipelines is as simple as stringing together shell commands. That's it. We'll actually run this example pipeline in the ``Hello World`` section.
 
-With Pypiper, simplicity is paramount. A user can start building useful pipelines using Pypiper in under 15 minutes. At the same time, using Pypiper provides immediately clear and significant advantages over a simple bash script.
 
 Who should use Pypiper?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+The target audience for pypiper is an individual student or researcher or lab
```

Comment:
Nice description of the overall objective/vision, and suggestive of motivation for design/feature direction. "Better than just writing a shell script" feels a little normative...in a good way.